platform/tests/EducationBenchmarking.Platform.ApiTests/EducationBenchmarking.Platform.ApiTests.csproj

```diff
@@ -17,7 +17,7 @@
         <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="8.0.0"/>
         <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="8.0.0"/>
         <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0"/>
-        <PackageReference Include="SpecFlow" Version="3.9.74"/>
+        <PackageReference Include="SpecFlow" Version="3.9.22" />
```

Comment:
Is there a reason why we've rolled back the SpecFlow version?

Reply:
updated it to latest along with specflow.xunit